cxplat/inc/cxplat_memory.h

```diff
@@ -113,4 +113,46 @@ cxplat_duplicate_string(_In_z_ const char* source);
 void
 cxplat_free_string(_Frees_ptr_opt_ _In_z_ const char* string);
 
+/**
+ * @brief Initialize a lookaside list.
+ *
+ * @param[out] lookaside The initialized lookaside list.
+ * @param[in] size Allocation size of each entry in the lookaside list.
+ * @param[in] tag Tag to use for the lookaside list.
+ * @param[in] pool_flags Flags to use for the lookaside list.
+ * @param[in] allocate Optional allocation callback.
+ * @param[in] free Optional free callback.
+ * @retval CXPLAT_STATUS_SUCCESS The operation was successful.
+ * @retval CXPLAT_STATUS_NO_MEMORY Unable to allocate resources for this lookaside list.
+ */
+_Must_inspect_result_ cxplat_status_t
+cxplat_initialize_lookaside_list(
```

Comment:
Not a change for this PR, but what do folks think about changing the function naming scheme to cxplat_<object>_<operation>, instead of what we currently seem to be using (cxplat_<operation>_<object>)?

Reply:
That would go against Microsoft API guidance, so I would argue against that.

Reply:
I didn't know Microsoft had API guidance on that kind of thing. Do you have a pointer I could read more on it? I would like to understand any reasoning for that guidance.